content/deserialize-custom-data.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Deserialize Custom Account Data
+title: Deserialize Program Data
```

Comment:
I was concerned that people would think a 'custom account' was a concept, versus an account used for program data. I *think* 'program data' is simpler than 'custom account data'. What do you think?

Reply:
Works for me! Valid concern given it's terminology they're not yet familiar with